src/PrestaShopBundle/Twig/Layout/MenuBuilder.php

```diff
@@ -66,6 +66,20 @@ public function getCurrentTab(): ?Tab
         return $tab;
     }
 
+    public function getCurrentTabLevel(): int
+    {
+        $currentTab = $this->getCurrentTab();
```

Comment:
Maybe `getCurrentTab` should store the tab value to avoid three different DB calls for this (at least, since the method is called in three different places)